docs/distributions/debian/installation.md

```diff
@@ -1,57 +1,59 @@
 # Introduction
 
-This guide shall help you in installing a Debian or Ubuntu based Linux distro, which is not available as a modified ISO with T2 support.
+This guide will help you in installing a Debian or Ubuntu-based Linux distro that does not have a modified ISO with T2 support.
 
-Some popular distros include :-
+Some popular distros include:
 
-1. [Linux Mint](https://linuxmint.com/)
-2. [Pop!_OS](https://pop.system76.com/)
-3. [elementary OS](https://elementary.io/)
-4. [Debian](https://www.debian.org/)
-5. [Zorin OS](https://zorin.com/)
-6. [Kali Linux](https://www.kali.org/)
-7. Other Ubuntu flavours like [Kubuntu](https://kubuntu.org/), [Ubuntu unity](https://ubuntuunity.org/) etc.
+* [Linux Mint](https://linuxmint.com/) (Ubuntu-based | Debian-based if using LMDE (Linux Mint Debian Edition))
```

Comment:
Two brackets together are not looking good imo. Maybe either use LMDE or Linux Mint Debian Edition